hevc_codec_registration.src.html

```diff
@@ -0,0 +1,212 @@
+    "publisher": "ITU",
+    "date": "August 2021"
+  },
+  "iso14496-15": {
+    "href": "https://www.iso.org/standard/74429.html",
+    "title": "ISO/IEC 14496-15:2019 Information technology — Coding of audio-visual objects — Part 15: Carriage of network abstraction layer (NAL) unit structured video in the ISO base media file format",
+    "publisher": "ISO",
+    "date": "September 2019"
+  }
+}
+</pre>
+
+Fully qualified codec strings {#fully-qualified-codec-strings}
+==============================================================
+
+The codec string begins with the prefix "hev1." or "hvc1.", with a suffix of
+four dot-separated fields as described in Section E.3 of [[iso14496-15]].
+
+EncodedVideoChunk data {#encodedvideochunk-data}
+================================================
+
+{{EncodedVideoChunk}} [=EncodedVideoChunk/[[internal data]]=] is expected to be
+an access unit as defined in [[ITU-T-REC-H.265]] section 7.4.2.4.
+
+NOTE: An access unit contains exactly one base layer coded picture.
+
+If the bitstream is in {{HevcBitstreamFormat/hevc}} format,
+[=EncodedVideoChunk/[[internal data]]=] is assumed to be in canonical format, as
+defined in [[iso14496-15]] section 8.3.2.
+
+If the bitstream is in {{HevcBitstreamFormat/annexb}} format,
+[=EncodedVideoChunk/[[internal data]]=] is assumed to be in in Annex B format,
+as defined in [[ITU-T-REC-H.264]] Annex B.
+
+NOTE: Since [=EncodedVideoChunk/[[internal data]]=] is inherently byte-aligned,
+    implementations are not required to recover byte-alignment.
+
+VideoDecoderConfig description {#videodecoderconfig-description}
+================================================================
+
+If the {{VideoDecoderConfig.description}} is present, it is assumed to be an
+`HEVCDecoderConfigurationRecord`, as defined by [[iso14496-15]], section
+8.3.3.1, and the bitstream is assumed to be in {{HevcBitstreamFormat/hevc}}
+format.
+
+NOTE: This format is commonly used in .MP4 files, where the player generally
+    has random access to the media data.
+
+If the {{VideoDecoderConfig.description}} is not present, the bitstream is
+assumed to be in {{HevcBitstreamFormat/annexb}} format.
+
+NOTE: "annexb" format is described in greater detail by [[ITU-T-REC-H.265]],
+    Annex B. This format is commonly used in live-streaming applications, where
+    including the VPS, SPS, and PPS data periodically allows users to easily
+    start from the middle of the stream.
+
+EncodedVideoChunk type {#encodedvideochunk-type}
+================================================
+
+If an {{EncodedVideoChunk}}'s [=EncodedVideoChunk/[[type]]=] is
+{{EncodedVideoChunkType/key}}, and the bitstream is in
+{{HevcBitstreamFormat/hevc}} format, then the {{EncodedVideoChunk}} is expected
+to contain a base layer primary coded picture that is an instantaneous decoding
+refresh (IDR), clean random access (CRA), or broken link access (BLA) picture.
```

Comment:
I wrote this to match the definition of "sync sample" in the ISO BMFF embedding, but I'm not familiar enough with decoder implementations to say for sure if that's the best definition.

Reply:
Also not familiar enough. Maybe Gary could weigh here.